cmd/ddev/cmd/config.go

```diff
@@ -368,6 +373,63 @@ func handleMainConfigArgs(cmd *cobra.Command, _ []string, app *ddevapp.DdevApp)
 		}
 	}
 
+	if cmd.Flag("update").Changed {
+		friendlyMsg := fmt.Sprintf("Update flag detected")
+		output.UserOut.Print(friendlyMsg)
+
+		detectedApptype := app.DetectAppType()
+		detectedApptypeMsg := fmt.Sprintf("Detected %s", detectedApptype)
+		output.UserOut.Print(detectedApptypeMsg)
+
+		// For the POC we need ConfigFileOverrideAction to omit app.ConfigExists() check
+		err = app.ConfigFileOverrideAction()
+
+		apptypeMsg := fmt.Sprintf("Apptype is %s", app.Type)
+		output.UserOut.Print(apptypeMsg)
+
+		if app.Type == nodeps.AppTypeDrupal {
+			// I can't call getDrupalVersion from drupal.go here?
```

Comment:
TIL lowercase/uppercase defines visibility. So needs to be renamed to GetDrupalVersion if we want this function to be exported. https://go.dev/tour/basics/3

Reply:
Your IDE can easily do this with a refactor action.